# HG changeset patch
# User John D
# Date 1481579304 18000
#      Mon Dec 12 16:48:24 2016 -0500
# Node ID 7cf0b9654d4471a7fd09e9c556b098611f933760
# Parent  9b096bffc10d330711471050d2054b3b56d5542d
cross-tools: enable std=c++11 for default cross g++ CXXFLAGS (Bug #49822)

* configure.ac: add -std=c++11 to default CROSS BUILD_CXXFLAGS

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1186,17 +1186,17 @@ AC_SUBST(XTRA_CXXFLAGS)
 ### the compilers that we use for building tools on the build system.
 ### For now, we assume that the only cross compiling we can do is
 ### with gcc on a Unixy system, but the dedicated hacker can override these.
 
 if test "$cross_compiling" = yes; then
   BUILD_CC="gcc"
   BUILD_CFLAGS="-O2 -g"
   BUILD_CXX="g++"
-  BUILD_CXXFLAGS="-O2 -g"
+  BUILD_CXXFLAGS="-O2 -g -std=c++11"
   BUILD_LDFLAGS=""
   BUILD_EXEEXT=""
 else
   BUILD_CC="${CC}"
   BUILD_CFLAGS="${CFLAGS}"
   BUILD_CXX="${CXX}"
   BUILD_CXXFLAGS="${CXXFLAGS}"
   BUILD_LDFLAGS="${LDFLAGS}"
